### PR TITLE
Ear 945 legal options design 2

### PR DESCRIPTION
**Pull request by farres1**
> BEFORE MAKING YOUR PR... There must be no linting errors, all tests must pass and PR is named after JIRA ticket number e.g. EAR-###.

### What is the context of this PR?

Implementing design changes to the legal options' radio labels.

### How to review

> Add to the list below as appropriate, including screenshots when necessary

- Import the **Capability examples** questionnaire from pre-prod Author into your local environment and:
  - ensure it can be opened in Author;
  - then, ensure it can be viewed in Runner by pressing the **view survey** button
- Does this require a migration? We need one if existing JSON schema properties change

### What to do after everything is green

1. - [ ] Bring this branch up-to-date with Origin/Master
2. - [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. - [ ] Click the **Merge** button on this pull request
4. - [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. - [ ] Move the Jira ticket for this task into the next stage of the process

### CODE REVIEW

File: eq-author/src/components/Forms/Input/index.js [hunk 1]
```diff
@@ -34,27 +34,32 @@ const radioButton = css`
...
+      height: 1.25em;
+      position: absolute;
       border-radius: 100%;
-      background: ${colors.lightGrey};
-    }
-    :checked:before {
-      background: ${colors.black};
-    }
+      background: ${colors.lightGrey}
+      border: 2px solid white;
```
Comment (MattElcock) [inline eq-author/src/components/Forms/Input/index.js:42]:
```suggestion
      border: 2px solid ${colors.white};
```

File: eq-author/src/components/Forms/Input/index.js [hunk 2]
```diff
@@ -34,27 +34,32 @@ const radioButton = css`
...
+      &:checked {
+        background: ${colors.black} !important;
+      }
+      :checked:before {
+        background: ${colors.black};
+      }
+      :hover {
+        border-radius: 100%;
+        outline: none;
+        border: 2px solid white;
```
Comment (MattElcock) [inline eq-author/src/components/Forms/Input/index.js:54]:
```suggestion
        border: 2px solid ${colors.white};
```

File: eq-author/src/components/Forms/Input/index.js [hunk 3]
```diff
@@ -34,27 +34,32 @@ const radioButton = css`
...
+        box-shadow: 0 0 0 1px ${colors.black};
+      }
+      :focus {
+        border-radius: 100%;
+        outline: none;
+        border: 2px solid white;
```
Comment (MattElcock) [inline eq-author/src/components/Forms/Input/index.js:60]:
```suggestion
        border: 2px solid ${colors.white};
```

**Review by MattElcock — CHANGES_REQUESTED** [review]

**Comment by jchapman68** [comment]
On my monitor the bullet point and paragraph alignment is different from the ticket.  The bullet should be beneath 'gal' and the paragraph should line up with 'a' from 'basis'.  There is also greater paragraph line spacing in the image on the ticket.  This may be ok on Windows machines but check with Joe.

**Comment by MattElcock** [comment]
> On my monitor the bullet point and paragraph alignment is different from the ticket. The bullet should be beneath 'gal' and the paragraph should line up with 'a' from 'basis'. There is also greater paragraph line spacing in the image on the ticket. This may be ok on Windows machines but check with Joe.

This is the same for me too. Otherwise, it works great for me and the code looks spot on. Well done!

**Comment by farres1** [comment]
> On my monitor the bullet point and paragraph alignment is different from the ticket. The bullet should be beneath 'gal' and the paragraph should line up with 'a' from 'basis'. There is also greater paragraph line spacing in the image on the ticket. This may be ok on Windows machines but check with Joe.

Looking more closely at it, it was the same on my screen too. I've changed the alignment to match the ticket's screenshot. I've discussed the change with Joe, and with that, I've edited the alignment.

**Review by jchapman68 — DISMISSED** [review]

**Review by MattElcock — APPROVED** [review]

**Review by jchapman68 — APPROVED** [review]

**Pull request merged**